pkgs/data/themes/catppuccin-kde/default.nix

```diff
@@ -1,24 +1,45 @@
 { lib
 , stdenvNoCC
 , fetchFromGitHub
+, flavour ? [ "mocha" "macchiato" "frappe" "latte" ]
```

Comment:
Weird minor problem, this should be named `flavours`, but if I call it that, then trying to refer to it in the installPhase instead refers to the packages `flavours`.